test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -725,7 +725,7 @@ dir -Recurse `
         It "Test member completion of a static method invocation" {
             $inputStr = '[powershell]::Create().'
             $res = TabExpansion2 -inputScript $inputStr -cursorColumn $inputStr.Length
-            $res.CompletionMatches | Should -HaveCount 32
+            $res.CompletionMatches | Should -HaveCount 33
```

Comment:
Probably should be 34 now, as you added `StopAsync`.

Reply:
Corrected. Thanks.